src/Counter.php

```diff
@@ -89,6 +82,10 @@ public function setTtlInSeconds(int $secondsTTL): void
      */
     public function getCacheKey(): string
     {
+        if ($this->id === null) {
+            throw new LogicException('The counter ID should be set.');
```

Comment:
I'm not sure it should be required.

Reply:
`getCacheKey()` can be call when id is `null`.

Reply:
Yes and that could be alright.